src/test/ensemble_tests.cpp

```diff
@@ -636,8 +636,8 @@ TEST_F(EnsembleFlowTest, CorrectPipelineDefinitionNodesValidation) {
 
     // Simulate reading from pipeline_config.json
     std::vector<NodeInfo> info{
-        {NodeKind::ENTRY, "request"},
-        {NodeKind::DL, "dummy_node", "dummy"},
+        {NodeKind::ENTRY, "request", "", std::nullopt, {{customPipelineInputName, customPipelineInputName}}},
```

Comment:
Not sure if it is worth it at this point but wouldn't it make more sense now to change the order of arguments for NodeInfo constructor? It seems that we almost always need to fill in the last part(outputNameAliases) but we skip (use defaults) for modelName, modelVersion.

Reply:
Maybe to do after CodeFreeze/Next release,